Traceback (most recent call last):
  File "/home/mohana/mhd_superhump/Athena_Analysis/./leaf11.py", line 26, in <module>
    simple_loop("Cyl_11", [0, 10000], 500, eccentricity_radial_profile)
  File "/home/mohana/mhd_superhump/Athena_Analysis/./leaf11.py", line 15, in simple_loop
    function(dname, fnum)
  File "/home/mohana/mhd_superhump/Athena_Analysis/branches/meccentricity_profile.py", line 527, in eccentricity_radial_profile
    aa = Athena_Analysis(filename=filename, grid_type= coordinates)
  File "/home/mohana/mhd_superhump/Athena_Analysis/branches/roots/athena_analysis.py", line 45, in __init__
    self.data = h5py.File(filename, 'r')
  File "/home/mohana/miniconda3/lib/python3.9/site-packages/h5py/_hl/files.py", line 507, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, fcpl, swmr=swmr)
  File "/home/mohana/miniconda3/lib/python3.9/site-packages/h5py/_hl/files.py", line 220, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5f.pyx", line 106, in h5py.h5f.open
FileNotFoundError: [Errno 2] Unable to open file (unable to open file: name = '/home/mohana/mhd_superhump/data_raw/globus_dump/Cyl_11/disk.out1.03500.athdf', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
